src/Jupyter/IQSharpEngine.cs

```diff
@@ -35,8 +43,11 @@ IServiceProvider services
 
             RegisterDisplayEncoder(new IQSharpSymbolToHtmlResultEncoder());
             RegisterDisplayEncoder(new IQSharpSymbolToTextResultEncoder());
+            RegisterDisplayEncoder(new StateVectorToHtmlResultEncoder());
+            RegisterDisplayEncoder(new StateVectorToTextResultEncoder());
             RegisterJsonEncoder(TupleConverters.Converters);
-            
+            RegisterJsonEncoder(new StringEnumConverter());
```

Comment:
What is this about? Would it make sense to instead add it to the list in `TupleConverters.Converters`?

Reply:
This is needed for enum-valued configuration options. I'll move it into `TupleConverters`, then.